src/main/webapp/itemPage.html

```diff
@@ -23,14 +23,18 @@
     <link rel="stylesheet" href="/css/user-page.css">
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBQKQXGIYnT1dc3c09KXic9SgBteMtb7No"></script> 
     <script src="/js/item_page_loader.js"></script>
+    <script src="/js/navigation-loader.js"></script>
   </head>
   <body onload="buildUI();">
+  	<nav>
+  	<a href="/index.html">Home.</a>
+  	</nav>
       <h1 id="item-header"> Header ...  </h1>
-      <h4 id="datesss"> Dates ...</h4>
+      <h4 style="font-family:'Proxima Nova Soft', 'Helvetica Neue', sans-serif;" id="datesss"> Dates ...</h4>
       <p id="description"> Description ...  </p>
-      <h1 id="item-pic"> Item Picture ...</h1>
+      <div id="item-pic"> Item Picture ...</div>
 
-  <p>map</p>
-  <div id="map"></div>
+  <!--  <p>map</p>
+  <div id="map"></div>-->
```

Comment:
You sure you still want the map commented?

Reply:
Oh yikes! I forgot about this. I will fix quickly